### ISSUE TITLE
Login does not work

### ISSUE THREAD

**Issue by Baliverbes** [dessalines/jerboa#1316]
### Jerboa Version

0.0.50

### Android Version + Phone

12.0.0.134(C431E3R2P2) ELE-L29

### Describe The Bug

Login does not work

### To Reproduce

1.Try logging out
2.Then log in

Expected outcome :
the login process comes through and I am taken back to the home page (publication list)

What happens :
the "Login" button reverts to its previous state, prompting a click. No other change on the screen I can notice (see captures). No amount of tries results in a successful login.

A few days ago I got the "User is being verified" message. It kept appearing for several days and prevented me from interacting with content, so I decided to try logging out then logging in again. I logged out, and that's when I was unable to log in again. 

The login and pass work fine on the web client. I copy my password from Firefox's password manager and paste it into the relevant field on the login screen, so I am fairly sure there is no error on that front. 
 
Here are two captures of the login screen. 
![Screenshot_20231226_111320_com jerboa](https://github.com/dessalines/jerboa/assets/13592760/f121c0d8-95aa-4f25-8925-8d0c8a68b3e1)
![Screenshot_20231226_111316_com jerboa](https://github.com/dessalines/jerboa/assets/13592760/e046e75b-47ab-429c-bbcf-d5249cff596d)

**Comment by MV-GH** [comment]
> 0.0.50

update to 0.0.54+, that version doesn't support lemmy 0.19 yet

**Comment by Baliverbes** [comment]
> > 0.0.50
> 
> update to 0.0.54+, that version doesn't support lemmy 0.19 yet

Thanks ! the play store does not offer updating the program. Is this normal ? should I be using another channel ?

**Comment by Baliverbes** [comment]
> 0.0.54 should have landed a week ago already on the play store (I have 0.0.55 installed from the play store)

Ok for anyone encountering the same issue, I figured out what is happening : apparently I installed Jerboa from an archive, not from the play store. The play store recognizes Jerboa and lists it as installed, but won't update it. Uninstalling and reinstalling (this time from the play store) worked fine and updated Jerboa to 0.0.55.

Thanks for your help and sorry for the noise
